### ISSUE TITLE
add <image> support 

### ISSUE THREAD

**Issue by codingBobo** [sammycage/lunasvg#113]
> such as this, the image block is lost

`<?xml version="1.0" encoding="UTF-8"?>
<svg width="280px" height="280px" viewBox="0 0 280 280" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <title>v2_detection@2x</title>
    <defs>
        <pattern id="pattern-1" width="100%" height="100%" patternUnits="objectBoundingBox">
            <use xlink:href="#image-2"></use>
        </pattern>
        <image id="image-2" width="170" height="170" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAKoAAACqCAYAAAA9dtSCAAAAAXNSR0IArs4c6QAAAERlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAA6ABAAMAAAABAAEAAKACAAQAAAABAAAAqqADAAQAAAABAAAAqgAAAADfcN5xAAAet0lEQVR4Ae1d3ZpcN26U/PkqF3udJ8lr5dmT22xWe4pNcEAc/JLsn7F15B6SQKEAEtXsGUm2f/7nf//Prx+rz3rkjx8rsSzmV4aA4d0tHsb947/+4aYbzkzeDIYIFax5TgqWaNSxiieS1TiK7+MfYl1b/qzBJ/RKLIv5+YMtJuJvsjjUwLHbCl8FiwRV/CjqzOTXr18/9oSKOnb0shN75gweLJ9Sx8k9XVzmbXo4j0l3QOAQKZ59oYLllY1mucJblWFR5vZzmm+7oAWCqniqeCppNY7ir5FECtMfx0S22sTVOLahv+w022wF99bbVKmn2iMuUsQ+btR3i6Wan+HDW7V6Qn9lfFVAVTzObiVGnLkUKdxfH/2s+SIuvzzBkc+WQ2ZryuJyWV+HOiAMtdhn8arJvoyaSOH9EipWJ5q1ylGNY/jftyqa9/WoH/uvEN5mDkuk2NksVFggACYCmMrPbnw5YRDwafUE5Tb3ZtMzKUzMSu6VGFaAJ1LA7kKl4N3mrsRXYxj+b3mrKuLYvk0VTpKEOa7EMLJIpIDaQoWXCQHL8rMSvxJTLswIeFXuzcYa1b/HvLmXjEixMV+oQLyqeci18mTry+KiGk7xRHkW/G+5TRfqpJCsSIH/I/WRudOcldiVmGszqb3QKX33cfMmO7L9jRoqIkWt7UZNNXhRPO1AVmIrMRXskQ49mWRDAFNlFZ4KFkmqeFZYVaQIHR/9EGso2B1B7MSyTUZTdw+ZGjKYqIg3+NWP/WwdVdFV8ayOFZEifAiVuNxGA7TTyGpsBV/B0ma/65gRSgazsv8N3lWRosybUGH8tmJF8dcT1v+A/WW+vvQ2XTy1HZEipSpUOMJmf+INlqkpg8EBvOvZuLHKJVdzVfG9oF2RgsYUKpxPE2tVLFV8pnZs8B3PYrNHqZn4DGYQJieLnNsiRd7r5QoVW/h2Yl0Q9a1VJzhupM8xLH/sV4RXwbJtHhFp5wuFChzE6gp2tbGrcewwvKlZ827e3Xiv6F3foqjMtIt8J0WK2h5/cTp58GbjwZTkAHR6KnFZbBY3FfIBi0gUwv+S23ThWE6LFCV83ajJ5j5FrJXDSNYZUp7iCRO9ECCErGbOYCiwgu0xWyJFPiPnl1CRKNm842JN5u1nkRs6p1urx/SMmrx8Rd/ybZrNYwjGC98WqUM+CxVANCjRJFcAifhbTZWYCvaW6JsaIuFEfmw7g6ng2FE+U6RIcxcqJU+I4a1ipTq9se/BrDOxR4/+W/m+sUhxzrZQ4U000hRBMh6w6UnkbPgsbiL/4IUnJOG7fewL/yt3uXyTouZC3b5QsWMIIhAFxGoKNohFituTjcngOsas75acGTL8DB5OC40JuSqAbN4srufeEmml/gsbC5UIE00zxZCIpTQvHz+5NuUwnnabfrBIcQx5oQKdaOoxsSZyoaRMTYQxa2tExS/Z+oq0N3hRQEvxxRyvvElpPzWhIgoNCppkCiKIo6LGmMVncYP4wyZFoYzqo7jID6IMZiS84P2/BcVM8RQ5inkkaV2oxBCI4+Vipbqssder1hXsxaJ8tf32sf/iApZFeqDOdaEiedBgVRSJuNu+gjwNn8HciBOGZ/EmUru3UHRDRX7kz2B6ne8UKercEyo2ETQSYlUFG8T18/kaMvgI4/k931cVb5uVbtOMADOYvtu3iRQ19jr3hYrNoMlBo02xBnGgP/2otZxO8ky+gsjUMgrxZZEycam5M0aFo/S3p8IcgehMgQRxI28Gl8EMwg+YWKKx7FHJUVzkZ/xLImXxS1Ojvq8b9VSDA563i7XXp9Zh1W7ZK50wGpChmD72PR7Pl0nEMC8XKWp36v8SKopEQ040JeBRRUL5MUbPiRqjHBn/p9SRqRUYRwic4i0i5QUo81moBAiERrBwdBr5dLE6uenNaNYQbuzJACaoY7cp4/SqL4kUnEleNWchXhcqsXrNJkw0OhymUJyYKN3kX+FZiZmSFhY7TS6kyYqpLNJKDRxbECiF+UIFCo3bbZ7DAbGqgs3kzGBop3LssWpuicV6J5fGl7AduU2Tb4aXijSxdwmJhUoRaNRus5x4VTAOnsoKa8pwDLIPmCSFNSqt4kfg1+QlIl24Rb8qXPkNfzR+p/lO7NPEynfM572WW16nRh7+svmqGBNxaZHuCC1RR3SW+RtVMu00E7FG/E00yGtgp5I8jOebSF64CJo3fexbZXkcnq/zlURq1eDZUUOiDo+CfOtCBQMEsCMCI/alYu01qDnplGg06iX39lhpqof1fL3Ip4oU+RM1pM/r4toTKmVCA1ebaMSpwjGwVMbR8WSuYtOm27QYmzmDp4s0U0QGg733/Z8RKiVdba4RB7HeBGtgqQT3DWPFWvZB6kx2YjltRZAe1vNd+VIiBUfAw0tv85WYG0k3KFzn/heTlBSNW2meE/cSsbayReFiSVt82bgiFqe4tEgdDtVVrVMluYyKQAn6uFFJJCcbQ5yUKTsaNRwVq1aLkVeDbtucxk4f+1oiK9ayd46niNQRlla6aUvw3D/6VwVmVbEiACPmmFgNfmzhlkPuy4mV0PRaE1nWhiQaliU/LtKEsFh6e1rguQuVaNGQU01Z4TJibkKKarT8mj1rozM6PIa36UK+UKQFsbT0wZsiVWIhJ84Ev2yhUkY0T2sg+SvjCpeSuyzWSo1tu0rSIscWXBODZkMSyw5X9C/iObG3+oGt4G8EPT7JQQIlmliohCSRnehhlUPBl8SqxLdtaXbNRmdwYpSN6ms0xnwsl2W/iI6L1CwucKBGeoVQkud9Y3mh8iRo5m5DqxwKHmKdBOvVZPkMu8trxPAjWp7fe6RTOThXpEnRtKQVrKyyEEvylBR8vSZUYkDDdptW5VDyuaKiWjEqsdw95lncCHAmjqCcqC+XFq/ZekQo0i9mf+bkcAMRl4zNCJRy7QmVWKpiozg+VsShYLfEqvChtImT1yrnRryE3da9oeNjP9ngG083HBFpQWhTHck4EufY80RiLC7uM0IlfjRstWngqMQr2ElY1TokPlrTnp85asLVbFcN2yJNCu223WQcCfQWbxmIt+/3Twu3ZedN7olKfBSfiQWW4SDW8W4VvlGDZR+AxGSFg9VJGUat0ifXCNBsMFs/3Rt4yj3GLG4EIClf6POxN92tWw3e83+EKtOjoXitPNk4gUvdrCKmlSdt13riWtmDFWM0xIJbwtgSKWpYqSOI2b09tTN43Ki8QUERGknKRjmq/Nk4gSOBtXe18I16Ya/UU8WPRMFE1hCtO922SIOyJresaXI+FqUbNMHHU/w5NRQeaiqhioQUZo7EX+XNxgkxYX/jAIWv1ShtyvrnL8ZhbqzuGHXx0OS5LIs0yT9KCvDqHkawmARcAj0tx/eoJFh4p+RoHD0biYhijKu8UkiDkE0EZlusjLq9kekcRB4Ou80pBg5rfgsS2O5XRco5kzwabNgCvkkjI0iZBDxKhGoaQuXet4k2uykSuYcXIgrFyg9Azi+u07eq2mi5H7m+6nq6SJWcdBxqzeTko8PBYZW5KlROwEUL+yiWxPIw8pD1OXFmNxrhhd8VK7A8r1xHu9LwnE+L9/yKryxShUMro9kc7Oi5GXw5nHgvLOsr/z4qmk2/RhI0iV7DuDGpckV4+PuD2sfDps3mrS/fiJW4QZiYXA1VG88bzeed8mkiRS4tX6sSlSpOvk0jnkNOzMMb1UsyGtdBbVO8icEePe7mq3IBb+VkPtQ9GsDsI6fFERZsADQ+buNzheImUg/v+YjbwYxzIaw2OvEa/IRtS6iyAC7cm2gB3tkgiTbi8HDMR7WOOjkvF6+Y375X5X55IHLNc0gfrQUmLVIRR3TT6GBCgTqxU45TC5HvqFB5jSQE2MYhkFAeRg7Pz4lDbORG4OGYuMbtymyNi6/5nBJpNvI5YzsLXrs1vziOiZTnYLWNvjDbNDXiJsypRZDraULl9XPRwt4OCI2mJyiSYNNI8VGsJShmN8U6JewLFqe5SzZeO59fJCmRiphbbsPvCtSIuXHvGBZy/Ml/thi5F4hGbGLChbst2oxgLQyzq2LlomTzgU3sdUCuM2171c5W2CaRCh/nG3M5UWLeKk6lHllytNZvVGogjz6QjNPRnIsWtl8/WSI2Jbw58pqtOMJIfxfhEGBft1zanGxy1IqTuQhj2EORGnGNVvGZAlWwVNr2+ARuXahapWiKfJ5QEBfutmit+khgfD/dFoq1xwwc58jOeV1s7oqU4W5pFN/LBKrkvtV3wJAXqpZMEy9wh4o3RZvNQfVp9Wi+bsNP9i0FbneKhQlza0QA+TCnuD5XP/YZZkmkLP6RRhhgxGOYH87i15NchdR7QrUSPfo8ezc3yEUL4tJtS/VoNWi+Lrh2azpiXbpVqQYasRf6+6TM1g5PrpsRATR5jOrtKTBzRHJ1giOZKoI9R6haVhIE920cBBduWrS8Bpm7i3OUZ4l1AK4Jj+FzwvQct9uU5S6JlMdJtSIn81MJpXE3vpSsBn6dULW60Fz5LBwWFy3EM24Yj4tyc4y09XX7TX66WWFDTB9vt663H54LNNpNKjCNjtnG3igP85EpPe7EppMcAF51vleo2h5ILNxXPNAh3ItrNNbioHzcL23XWhUragSWx/K6aQ4/YfqYEinFtPBpQcz5kYXng16ATNb1eULVzoaEw33JDaZFSzk4LxfhNb+JFfVc+HarDiXyItkbpWPbgJuU5+FzjuMgiZnT3FdV/J3hjOVQHd9DqNqRkbC4LzgULlqE3b53hJHzgo/Wfa6KFXF4eH7M5QsmLlKJB8f1jE+Bx6LZ3C+cxwUedr4wr/4nU9F+XlhgVMrkJ1FNxmth1NuE22OGODiW+GCjeZs+FuPjm/LBzOLHG6HbVJGSbw4kRn1kOXTAAesrchTKXLtRWdPSud65ca1eUY9721I8YkiM18g/8tucfjiiQ2E5hkjJ1kf1DULxNFIMrXfH03y79STi14SaIL5BqNk3h2N45oFq9bB8t9sWPiWGi7X5gbmwXJi//tWJMcBHt2c3Tyeg2SZAsNiND+jf5X6dUFd2qAgjpNlplJbv4ruJlhdxxbTvWy8x/vzJ/kI26rheTaQ0f6iURzfMbHBW4PmbPp8t1JWmaGKLeDwBCL4mWuK7fO3mhFivX+2m/NflBN81/vr/a4J/+LcEVi7LTrn+5uNfT6grDRViDCmA78LCLdoeDP3fQPv1z18//vV/UOrDNca+/D3UT+C3UKMzs0Qs7BBse/3Hzx///N9//vh5/ac92ves4BfYKOVv//0Evq9Q39D8cXvyc0Qd/fXzj0usl0Bxo7bnumFbDNVKI4+nuecjzN94PC/Ub37g7XtQbw/ch/n1IjFCqFi370mvT/72fSv+jgBhoN9rPj20pm8TJqdYEFaY/w7Ltd/w/+YnE4qR70+Kg6+vOYm0CZDWEF37FvX6nQCI93rGb0ld85YfRrikQB9weO+PxEqEFyux32x9/kb9kAOYxFCtSWs4t13zIXbYscYPVZjz5xIWbtdRS/c10QLe/4J2M1OsFCPZOa81l7EarsKnxb/J9q2FOgSwe/hWvLRf6yFQNAx+2EikDN8ECjvEev0atVJct7d4fCGRMQ5Ah70t+heJ4b5oTnk83A6/x7vh+3ih3hq8sdkRGjVC+vtaEyk0Nl4jQZ8wUTSxduEOGGItwQIk66BAxttMFo7w1VHyy/jT+SS/sv4IoT5FjHKz0eFq/m6bBApe2LlP5kKjJR81H3byC9v4loCCyQ9+yQcbPRwHm4elmJ1R5tO4DtfwMqFOYsTGDm/kdlZZfg3HbJNIyc5HmssCZDMvXLtV8X0pYuCnWDEnwYJynBvnozgAtIdj4Y/wGseuTdag8RXqOirUcahUVKEQClkeK7k8bPdNAkVRMuZaj/3CJ/2XqQmTHL1x+Pe72g9RwFMz5fxykY/+fbCRi/mIGib3oTwcpNTL3S+Za3XxxKzGslCnAyNSRkiml4zVvB6e+VyRdtwNc224/VClbRwNIf5rPokVeOEfWPIBgt+PvZ7p/B+mZp9iHhb/K48Fkurzo17rZTWqQp0Og5f27s2s5I9imH/sm9mmBpKdRpwNn/OzojkOmzA0v8ZQrIinOMx701TBMj+mU1wzJL50/oHkuYfxfZPxP5sYJXxYgUuHjs1E+xB+7YacOAh/jSp2HOA8GR//JFJyR2IFTsaQDS7thoUfDxcd1f3w5L9yDkSt8uQzusjP+5Op3QOJ4oVfvUVlY1jMwPNjZf6poZbQGH76AQucMoYEw2Ja6m4nwcKm1kbxD0ALXfrCeYhA1kT2J4zqR/8T8tiUJzab4RCY0VRhb4VymzYnG4327h4eNBlYY1TFikjOT7Gw84cEdGHBg2fsjeMwZ1jpWloTHwXzesl2aHyPUE9tKMOjYEYjFd8kDu6/5iOuePjj41+LQ7MpD58Dq61hJzzm9JBoLl9asIjVuIizOlINFHeQ+zVCPVhw6WBF3klownfj5f5rPmLJbo3UpGjkIrzm7YcrFMHt4JBry0b5gMfTqB6LUfvDM38lfI+ZnZsrzk1UdG60To7PEepiMW7NFU4FO5ql+FyR8qK02O4f/BzP52ga4mnkvj4fN6/EyDXwsOGxamL+8IZ9MH1xeryEXR2pLoq36id/H88JNZlQ5PeXVU4FPwlI8ZuNpsqumImD7NlRExnFcl+fl8QKHmq8tjfhJ8HCHO6JeB9gfH3Ow/Mgg7GP8v9nalQLQv4ajgMT4s1SGfjRDMOvHgqw9FzziYPZaZoduUiGuBDMG9XnA8t9hJU2KsCycz/DIMfIQxhrRByLtWBH7JSLxk6av1F5A49UJEhW+I2YIS6kMDCq3cKKUsfSw+OgPf8gEZMeBxG1fWg8mg00sOPx8goMiXU6swfL/SvFRjnukeuWntO/UbFheq2n8iNX+J2YceAORm0k8Py51hMX+SSO7JmRN9qaMx4S0RAg86k28nNusskRGIZDrpFPYrU1xTMODXbKNt+oO02oVLSTx4gdokIdBqaVqPmkTa4re3OwEMJUp4VF82UNWRtxkoAkD/lpFDgSa6pOyYF1lI9iiuMfjRjkT0ow1bOTx4jFgU6H6u1D80lbXw9O6Z821BcRhsQgY7mdz4HraxJOC5UYhmt+7YsWY+EYFnmn3FqMZgMH49EgK7b5Rl1hiGKiJm7EDzGBI8qj+TVbozIdX9UyyFTHF0KfoYksdgJJX19DMCOHxIAANjwer+dvwf2L4OJiHTVwvDUnHvituqxYxf48oe4W58TfDszBmoekxUibXCsH+CpTKFYUoomYFxj5JZbW/RxItLfzJ5w1HhCt/8OUldiyY0P0sjCRPYifDinAroh04ue1IteBh5o9qHgTYXTWU6zEEaFl5/4IQ1gagWcxqGOqhXCZkbgYXybsjFAjwWQqCTggoElEkXAsv2W/akzzOxyZrfKmV/EpgWREkMHI4khk3b4lWHAQX6KWdaGiWfTqhS8PQeNvAgrwra5KMRFfhcvCes2QPrkGJ7MNsTLbLS18np84I8yNuPOyuG3BJmqpC/WUOFFcwPW4Q5mK2BTh6uNhNF+33d4MnFyLI7/nI4wYh9CEfVoyIUz2vhgcAS4UK/jAEfH0vNMg4kiwo7YJnFwQp6gnJ1Q0g17JfC4swTUJB2QZQXgYxzflcnDunqpO0YiUUGQM5bTsWX8VR3gakV/UQKIlyNJIvNfoCxVNO9m4BJ96i2Zq8DCWz7LzU5UYsZ5EzuNOzEXzGyWzTTcXs6upIz8FAZfFUgyNSuwRwV78d6GiEfSiAnbHBN9NoMgpRGGW4eEsX7dPQrOwZuJ1xyQyoskKhOEmHmYnymmM/BxcwfI4zBEr4ncF+yVUNOl0o5Kck1iw0WRciLP20+23vMjNHyueY7y5aNYNGvkRYGGYvSxWFnuriRuAy2J5HM2V+FXBfv0RKpGfGhNNftotij0k8k9breKn4ORitemJuJJYUW6Cc+yqgh1BbIJ4wUGCnepmIXL6daNKz+oaDU80Xb3NEnGtrAjn+btPzR/t2eONYrN+0VA3TGCnpgufypPBUCCwFTzF8dHgINFyqJyfEyqamGikeYsmYlvxEc7zWz7Nrtnk6WUwMoatm7Cyzbdwwv5UsaJ2kY9tJz8Fh8LjCXZfqGhWsmHqLZaMbadQwTrHptbh4F/uUpro1uDhPR+RZjCExQh8NYbH09zg0QS7LtSiQG/iKMS3fWVE6mG6T62DDo5GjUex3bgo/lljUhzTrYpaMnGGaNytZHhdgu40cnPB1oWKhilNs+pRm1mIb7wZvIfxfFbhK/ZMHtHcJipha6k1m1eTwC+JFfyCx0vZfMBXYyxSgwt7qQk104heBAR6EyniCxyNKoPPYFpqARTLXvpnD54ohO9lYsWJGSJbOkyFKydUNDTZVFWgqDYZPzaWzRnxdr/6phnJ2CTiY9CnT4XwRj7LDoDwvVSsSv5R88qECdYXalYsvYibGKi4avOz+AgX+am+aNR4NFvE8yb/lliF8FNbYAJL4SPQxacLFU0oNMK9RQs8rd4sPsIx/+0NxHzTGVn2CXR+0YRkCaJqR3lKzLJYDb7UKSh1pOIU0CxUNKrYrJsIKEmRp4VlY7K4tp0CmGp/9niwgWapSo63iVWpxazbcDyEil4W+3n0FkVx2fwZnIfxfNohGXjzDapxPMsWCSDyo64MhuqvYCmGxp3Yi+PxZ/1ElhzNJhlNDWlX4zRixmXWGcRp7mVbskFLH/8oKslP9d9u1SpHMR/lbSNiF+Pnj/6J9b44fosiBRPWPaOwFLCqSAvxIvP3XSrCOCJWhTd9SAuxaaGqjUdlO82vxGawGYx2mqtxGtezbF5zPR/qUfzbYjV409tHTUpdVnwoVPMWBeNqgxFXic1gGUZ9UzG/dRg3uxVj2W8EjkFpkvvx71ClXFY+GazgJGRaV/FT8LVIxptCDQW62qxqXAafwcgD+o5rr6mez9nr229W1Ibag/pVoao3Em12RxTV2Cr+qlGt3ePxfLTn7zIGzbbE8BFixRk7gp2E+rRbFEVUBZHFM1xZpKhr4VHzLPCYIZ7gPJ9JyByV+AoWKYCvxrDSxlThGEJ1D5+JYZBVJtX4LD6Ls2r14j2fxXfArt5uFV6lybdwBbOdlydR+Lk7NQcH4/nDvUXBuNuwanwV33etvtEWuTrl8wfWiOcnExmU3KpYFZxg0percZKt84wbVfqbQHcbvRt/K4oZGLcqUgY9OmV5j/JKMq/Rng88kV/mYutPFasu1BPNWOHIxmRwEcbzez7W1GdNVbFUk2XEamDU/AY2LAtxq7GMfBYqGrTbpFWOxbwvvU3Zwb19eqD5bQ8Gz1GxIpGRJ3uOX0JdFMqUaJWjEsewpkgZZqqPFpGfcO8eN5ubFoeR55PEeu4/QLHa/EpcBfskkZlvDi+fIYQW4vk8Tvh2YiPu7v8UsX7dqMnCVdiqgCpxAmsKRuDUej3jbrzHXfCpAinED2hWzFkcEVfxFIdxIXZfqKuNXY279mmKlB+GNd/Ia1G+1Z5pegaDTRg4801j4FPngdhC/J5QV5tejcvis7jUSSqgZ/MrKSvN1MLLNkM8TxErijPyybrXhbratGqcwJu3qcDJjbZ1BqMGvsdoimOlnKQgPGqznl3uRPyaUL9Zw73Dn3zv2leiUVOdcrEbX+B7l1jrQt1pZjVW4LduU9mM02tR61H6U0Ks8DjYd4g1L1Q0YqcZ1ViBN0WaVYTgy4Zx3HYNnCw5N0Uh4x1hTdAsDkEO1qzLiZnqsBaIVzhyQt1t8m68tSnYT3Gf4vFq/bv4FKGVty44YqHuNnAlXsRs32SCr3xoCDjBsZS4B4nGqVQZDAKzuABr3qpBHNyph9XpC3W3OSvxIsYVqcCmNv8bVD8BJhgZ/Cqx2kLdFcFK/EqMPLnvul4Vg9yvwzNBszgKcvCvEKsu1F3B7Mb3wzlym2ZqyWCoYe8cHbEslVXlc/DPFutdqLtNW40Xca5Il7ry5iCnyccre2UuVvwzxToLVYiF1ZCbrsZX47L4LC6xu3e/cVwRJOo3IVVRB3i3ziDWrPFyPISKhh5sqpcw43NF8Y4635HTOqiNZluUZXtQwzPEuvQfSVM3ttpMEeeKVE28aRT5N9n2wgMBlMgrXBUsFbESsxE7f/QTUXVcbXY1roKvYKv7/Y0PT8C9VRFdFPq+UFcFocS9/DYNj/uzAKP52SZncQvCaScT8I96rWMM4nnYnlAVsXHyo/NKriw2izu6kQ8mKwhn7CKIOSXWdaHuNFmJ/X2bjtafnQRCOpIsyHFCrGtCVYSW3rASG4pUiUnnOwAM6zuQY1DsNn0QLU6C/Ius17eke8R1ob5ZNOFBna7vNF+4gSSg0vcKNpn+BkvkcMUaxNeEuts0JT68rZSY2yGtGJ7Fu1LLp8UEojHLXY0jQie+JlQiXBkVYYQireZRclQpvgPevZlObcARjZsiiAtrN+LzQn2HCN6R0+3ChzmNpqpVVrAqwTnjilj/DU0U/2taNlHFAAAAAElFTkSuQmCC"></image>
    </defs>
    <g id="main" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="check" transform="translate(-235.000000, -220.000000)">
            <g transform="translate(235.000000, 220.000000)" id="v2_detection">
                <g transform="translate(40.000000, 40.000000)">
                    <circle id="Oval" stroke="#FFFFFF" stroke-width="5" cx="100" cy="100" r="100"></circle>
                    <circle id="specialCircularSymbol" fill="url(#pattern-1)" cx="100" cy="100" r="85"></circle>
                    <circle id="Oval" stroke="#FFFFFF" stroke-width="1.85714286" opacity="0.300000012" cx="100" cy="100" r="65"></circle>
                    <circle id="Oval" stroke="#FFFFFF" stroke-width="2" opacity="0.600000024" cx="100" cy="100" r="40"></circle>
                    <circle id="Oval" fill="#FFFFFF" cx="100" cy="100" r="9"></circle>
                    <line x1="100.111111" y1="100" x2="100.111111" y2="0" id="Line-2" stroke="#FFFFFF" stroke-width="2" stroke-linecap="round"></line>
                </g>
            </g>
        </g>
    </g>
</svg>`

**Comment by ivantishchenko** [comment]
@sammycage 

I can take care of this. I already have the parsing of image tags.

The drawing of raster texture will be performed with `plutovg_image` once this is merged into plutovg.
https://github.com/sammycage/plutovg/pull/30

Once the above is merged the only thing left is to decode an image and draw the bitmap with `plutovg_image` inside a `layout()` method of `ImageElement`.

**Comment by ivantishchenko** [comment]
@sammycage 

what would be the best way to proceed? 

I think it would make the most sense to merge https://github.com/sammycage/plutovg/pull/30 first and then use `plutovg` as a submodule in `lunasvg` such that `plutovg_image` becomes accessible from `lunasvg`. 

Alternatively, we could just apply the changes with `plutovg_image` to the local copy of `plutovg` inside `lunasvg` as one single commit.

**Comment by sudoLife** [comment]
@ivantishchenko @sammycage any news on this? Great work on the library, and this feature would be a very appreciated feature as well :)

**Comment by sammycage** [comment]
@sudoLife text and image will be implemented in the next major release.

**Comment by sammycage** [comment]
@codingBobo Support for `<image>` elements has been added in the latest version. Please update to the newest release to use this feature.
